### ISSUE TITLE
Lexer Error when using grouping and cellNav and clicking group header label

### ISSUE THREAD

**Issue by MatejQ** [angular-ui/ui-grid#6073]
If you set up ui-grid this way:
- use cellNav
- use grouping
- group on a column that contains data with special characters in it (for example diacritics)
And if you click a group header for group with special characters, you get an exception (Lexer Error).

```
Error: [$parse:lexerr] Lexer Error: Unexpected next character  at columns 3-3 [ú] in expression [Matúš Miček1 (250)].
http://errors.angularjs.org/1.5.0/$parse/lexerr?p0=Unexpected%20next%20character%20&p1=s%203-3%20%5B%C3%BA%5D&p2=Mat%C3%BA%C5%A1%20Mi%C4%8Dek1%20(250)
   at Lexer.prototype.throwError (http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js:13458:5)
   at Lexer.prototype.lex (http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js:13417:11)
   at AST.prototype.ast (http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js:13581:5)
   at ASTCompiler.prototype.compile (http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js:14040:5)
   at Parser.prototype.parse (http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js:14927:5)
   at $parse (http://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular.js:15035:13)
   at getCellDisplayValue (http://ui-grid.info/release/ui-grid.js:5617:9)
   at Anonymous function (http://ui-grid.info/release/ui-grid.js:15626:21)
```

You can see [this example plunkr](http://plnkr.co/edit/BkA0E05GUMIkrSSd3oE6?p=preview). Click any of the two labels "Matúš Miček" and observe exception in console.

The problem is probably in method Grid.prototype.getCellDisplayValue (line 5616) where it passes rendered value to $parse which then fails.
Hopefully someone can fix this, in the meantime I have yet to find a workaround.

**Comment by mportuga** [comment]
@MatejQ I can't seem to reproduce that issue. Perhaps it has been resolved in the latest version? Could you verify and re-open this if you can reproduce it. Also, if you can, please mention which browser and OS you are using.

**Comment by MatejQ** [comment]
Strange... now I cannot reproduce this either (using the plunkr). It was in IE11, Windows 7. It works OK also in Firefox now. I will report back if I am able to reproduce it again.
